nodes/clarify_util.js

```diff
@@ -95,13 +97,17 @@ module.exports = {
     validateString(validationErrors, 'type', type);
     validateString(validationErrors, 'description', description);
     validateMapStringWithStrings(validationErrors, 'labels', labels);
-    validateMapStringWithStrings(validationErrors, 'annotations', annotations);
+    validateMapStringWithString(validationErrors, 'annotations', annotations);
     validateMapIntWithString(validationErrors, 'enumValues', enumValues);
     validateString(validationErrors, 'engUnit', engUnit);
     validateString(validationErrors, 'sourceType', sourceType);
```

Comment:
Same here, can we validate the allowed values?

Reply:
Allowed values are a string. So the input to the function would be `["*"]`.  For me, it only adds complexity.

Reply:
Allowed values fro `sourceType` is "measurement", "prediction" or "aggregation".

PS! We should never add `["*"]`; in that case (with my original proposal), you would _omit_ the parameter all together. But I really don't mind if there is one function with an optional parameter, or two functions with different names (as mentioned in the other comment). Do what ever you think works best for you.

Reply:
I think we're not talking about the same thing. This function is validating `annotations` which is a `map[sting]string`. The key must match a certain pattern, and the string value can be whatever. `sourceType` is validated with the function `validateString`.

Reply:
Right I see the confusion.

The comment was made on line 101 / 103 (`validateString(validationErrors, 'sourceType', sourceType)`), not line 98 / 100.

Reply:
`validateMapStringWithString` is fine as is!

Reply:
We don't have a use-case for validating any other map then labels and annotations.